### ISSUE TITLE
scrolling with mouse isn't working

### ISSUE THREAD

**Issue by Sinnlosvoll** [KaOS-Community-Packages/tilda#1]
works fine with shift+pgUp/pgDn

maybe missing library?

**Comment by bvaudour** [comment]
Indeed. Did you read the instructions of compilation?
https://github.com/lanoxx/tilda/blob/master/HACKING.md
You need also to add gtk3 and glib2.

Concerning the README.md, don’t forget to add the URL of the website and the URL of the kcp depends (vte290 & confuse, here), as said in http://kaosx.us/kcp-guide/#Formatting_standards_for_KCP

This makes us easier to know more about the package without need to have to read the PKGBUILD.

**Comment by Sinnlosvoll** [comment]
Yeah, I read the hacking.md and compared it to arch's pkgbuild and there are no differences, except scrolling working on arch (I come from there, if that wasn't obvious).

Also dependencies are added now and readme should be formatted right.

**Comment by bvaudour** [comment]
Thanks.
No idea how it works on your Arch, but don’t forget Gtk3 isn’t installed by default on KaOS, though.

**Comment by Sinnlosvoll** [comment]
Yeah, I forgot. It was my first PKGBUILD and I just "solved" obstacles along the way. 
Thanks for catching that. While I have you here, could you also please look at #2 and verify it on your system. It seems weird to me that the version inside the repos are different than what I get when locally compiling.
